__tests__/nc-games.test.js

```diff
@@ -30,4 +30,35 @@ describe('GET api/categories', () => {
                 })
             });
     });
-})
\ No newline at end of file
+})
+
+describe.only('GET api/reviews/:review_id', () => {
+    test("returns a review object with properties", () => {
+        return request(app) 
+        .get("/api/reviews/1")
+            .expect(200)
+            .then((res) => {
+                    expect(res.body.review[0]).toEqual(
```

Comment:
the review should come back as an object, not an array of one object, so `{ review: { review_id: ... } }`